### PR TITLE
Fix:ASC-00000 - update user profile

### PR DESCRIPTION
**Pull request by NaingAmity**

### CODE REVIEW

File: src/screens/EditProfile/EditProfile.tsx [hunk 1]
```diff
@@ -89,29 +90,63 @@ export const EditProfile: React.FC<EditProfileProps> = ({
...
-    return `https://api.${apiRegion}.amity.co/api/v3/files/${fileId}/download?size=medium`;
-  };
-  useEffect(() => {
-    setDisplayName(user.displayName);
-    if (user.avatarFileId) {
-      setImageUri(avatarFileURL(user.avatarFileId));
-    } else if (user.avatarCustomUrl) {
-      setImageUri(user.avatarCustomUrl);
-    }
+  console.log('old avatarId', user.avatarFileId);
```
Comment (topAmity) [inline src/screens/EditProfile/EditProfile.tsx:93]: remove console.log

File: src/screens/EditProfile/EditProfile.tsx [hunk 2]
```diff
@@ -89,29 +90,63 @@ export const EditProfile: React.FC<EditProfileProps> = ({
...
+
       const { data: updatedUser } = await UserRepository.updateUser(
         user.userId,
-        {
-          displayName: displayNameRef.current,
-          description: aboutRef.current,
-        }
+        userDetail
       );
       console.log('Update user success ' + JSON.stringify(updatedUser));
```
Comment (topAmity) [inline src/screens/EditProfile/EditProfile.tsx:151]: let remove this log too

File: src/screens/EditProfile/EditProfile.tsx (hunk 1 above)
Comment (NaingAmity) [reply src/screens/EditProfile/EditProfile.tsx]: removed console !

File: src/screens/EditProfile/EditProfile.tsx (hunk 2 above)
Comment (NaingAmity) [reply src/screens/EditProfile/EditProfile.tsx]: removed console !

**Pull request merged**